blivet/util.py

```diff
@@ -1160,7 +1160,7 @@ def detect_virt():
     except (safe_dbus.DBusCallError, safe_dbus.DBusPropertyError):
         return False
     else:
-        return vm[0] in ('qemu', 'kvm', 'xen')
+        return vm[0] in ('qemu', 'kvm', 'xen', 'microsoft')
```

Comment:
What does this actually do? Enable running tests under WSL? Could be useful to know, perhaps in the commit message :)